src/main/java/by/exadel/internship/config/WebSecurityConfig.java

```diff
@@ -24,13 +24,19 @@
 @EnableGlobalMethodSecurity(prePostEnabled = true)
 public class WebSecurityConfig extends WebSecurityConfigurerAdapter {
 
-    private static final String[] AUTH_WHITELIST = {
+    private static final String[] SWAGGER_WHITELIST = {
             "/swagger-resources/**",
             "/swagger-ui.html",
             "/swagger-ui/**",
             "/v3/api-docs",
     };
 
+    private static final String[] WHITELIST = {
+            "/auth/**",
+            "/internships",
```

Comment:
Well, my proposal was incomplete. I don't like that this url can allow unauthorized access to some endpoints which should be secured. Based on current implementation security I offer to create specific url path for public endpoints. As far as I know there're 3:
- GET /internships
- GET /internships/{{internshipId}}
- POST /forms
May be, add `/public` in the beginning? in WebSecurityConfig you can specify only `/public/**`
@SergeyKoval @viktarkolbik what your suggestions?

Reply:
In this case it should be separate controller which will mix public urls for different entities?

Reply:
If we want to add "/public" i think we need another controller

Reply:
separate controller which will contain all entities or public controller for each entity. And probably the second option will be better because in #36 PR additional public endpoints will be provided (countries/cities).

Reply:
Fixed

Reply:
Ekaterina and I came to the conclusion that this case would be better.